spec/security_considerations.md

```diff
@@ -38,3 +54,80 @@ be valid. This has two meanings, both of which are required:
 
 If a URL of a DID document or [[ref: KERI event streams]] results in a redirect, each URL MUST satisfy the same security
 requirements. A URL MAY come from a URL shortener.
```

Comment:
I think I was the one who wrote that a URL may come from a URL shortener. This was true when I wrote it, but has become false now, I think, because URL shorteners won't preserve the AID at the end of a URL. So I think we have to remove the sentence.

Reply:
Great catch! Fixed in my latest commit.